src/include/device_factory.h

```
@@ -2,10 +2,10 @@
 
 #include "smart_device.h"
 #include <QJsonObject>
-
+#include <QObject>
 #include <QPointer>
 
 namespace DeviceFactory {
```

Clang (14.0.6)

[warning] invalid case style for variable 'DeviceFactory'

(readability-identifier-naming)